src/test/suite/format/samples/ahk-explorer.out.ahk

```diff
@@ -3493,25 +3493,25 @@ return
 #if winactive("renamingWinTitle ahk_class AutoHotkeyGUI")
 
 $esc::
-    if (focused="flistView") {
-        if (canRename) {
-            canRename:=false
-            ; gui, renameSimple:Default
-            ; gui, submit
-            gui, main:Default
-            ControlFocus,, % "ahk_id " ListviewHwnd%whichSide%
+if (focused="flistView") {
+    if (canRename) {
+        canRename:=false
+        ; gui, renameSimple:Default
+        ; gui, submit
+        gui, main:Default
+        ControlFocus,, % "ahk_id " ListviewHwnd%whichSide%
 
-            gui, renameSimple:Default
-            gui, destroy
-        }
-        return
+        gui, renameSimple:Default
+        gui, destroy
     }
-    send, {enter}
+    return
+}
+send, {enter}
 return
 
 #if winactive("create_folder ahk_class AutoHotkeyGUI")
 
-$enter::
+    $enter::
```

Comment:
This label is indented because it comes after an `#if` directive, right? I thought we discussed that this is not intended behavior, as `#if` directives apply for all code below them. If this `$enter::` label is indented, shouldn't `$+enter::` and `$^+enter::` (lines 3519 and 3520) also be indented?

Reply:
Formatter detects `#if` directive like `if` command. #159 fix this. When I merge `main` branch it must gone away.

Reply:
Now `#if` directive detects like `if` command and formatter apply rules for `one command code` scenario. When this PR and
#181 will be merged to `main` branch this wrong behavior eliminates.

Reply:
Alright, we've now merged #181, let's update this PR to reflect those changes.